doc/ruby-4-upgrade-check.md

```
@@ -0,0 +1,82 @@
+# Ruby 4.0 アップグレード前の破壊的変更チェック
```

markdownlint-cli2 (0.20.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)